org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/thingTranslator.js

```diff
@@ -115,7 +117,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
     // Quirks:
     // - Nest does not have an external temperature field, so that is left out.
     // - Away Mode is not implemented at this time.
-    
+
     // Get temperature scale
     var ts = providerSchema['temperature_scale'] !== undefined ? providerSchema['temperature_scale'].toLowerCase() : undefined;
```

Comment:
providerSchema['temperature_scale'] could be null. Check null or undefined

Reply:
Fixed.